dotCMS/src/main/java/com/dotmarketing/util/Constants.java

```diff
@@ -134,4 +134,7 @@
 
 	// REGEX to validate emails
 	public static final String REG_EX_EMAIL = "^[a-zA-Z0-9_!#$%&'*+/=?`{|}~^.-]+@[a-zA-Z0-9.-]+$";
+
+	public static final String CONFIG_DISPLAY_NOT_EXISTING_USER_AT_RECOVER_PASSWORD = "DISPLAY_NOT_EXISTING_USER_AT_RECOVER_PASSWORD";

```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') 'PASSWORD' detected in this expression, review this potentially hardcoded credential. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2068)